### ISSUE TITLE
Module currency-symbols not found 

### ISSUE THREAD

**Issue by Kpython2022v2** [snoopyjc/xls2xlsx#16]
* xls2xlsx version: 0.1.5
* Python version: 3.10
* Operating System: Windows

### Description
I am simply trying to import the package but getting an error for currency-symbols

### What I Did

```
ran the below code
 import xls2xlsx (even tried from xls2xlsx import XLS2XLSX)

Traceback (most recent call last):
  File "c:\Sample_project\venv\defra_excel_read_v1.py", line 13, in <module>
    import xls2xlsx
  File "C:\Sample_project\venv\lib\site-packages\xls2xlsx\__init__.py", line 3, in <module>
    from .htmlxls2xlsx import HTMLXLS2XLSX
  File "C:\Sample_project\venv\lib\site-packages\xls2xlsx\htmlxls2xlsx.py", line 40, in <module>
    currency_symbols_constants = importlib.import_module('currency-symbols.constants')
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.1520.0_x64__qbz5n2kfra8p0\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ModuleNotFoundError: No module named 'currency-symbols'
```

**Comment by Anas-jaf** [comment]
check this solution https://stackoverflow.com/questions/72732983/why-do-i-get-no-module-named-currency-symbols-when-importing-xls2xlsx-even-t
which need you to go to the module and change in it underscore sign and you are good

**Comment by snoopyjc** [comment]
Fixed in v0.2.0
